packages/e2e-test-utils-playwright/src/editor/index.ts

```diff
@@ -52,15 +53,15 @@ export class Editor {
 
 		return frame;
 	}
-
-	clickBlockOptionsMenuItem = clickBlockOptionsMenuItem;
-	clickBlockToolbarButton = clickBlockToolbarButton;
-	getEditedPostContent = getEditedPostContent;
-	insertBlock = insertBlock;
-	openDocumentSettingsSidebar = openDocumentSettingsSidebar;
-	openPreviewPage = openPreviewPage;
-	publishPost = publishPost;
-	saveSiteEditorEntities = saveSiteEditorEntities;
-	selectBlocks = selectBlocks;
-	showBlockToolbar = showBlockToolbar;
+	clickBlockOptionsMenuItem = clickBlockOptionsMenuItem.bind( this );
+	clickBlockToolbarButton = clickBlockToolbarButton.bind( this );
+	getEditedPostContent = getEditedPostContent.bind( this );
+	insertBlock = insertBlock.bind( this );
+	openDocumentSettingsSidebar = openDocumentSettingsSidebar.bind( this );
+	openPreviewPage = openPreviewPage.bind( this );
+	publishPost = publishPost.bind( this );
+	saveSiteEditorEntities = saveSiteEditorEntities.bind( this );
+	selectBlocks = selectBlocks.bind( this );
+	showBlockToolbar = showBlockToolbar.bind( this );
+	transformBlockTo = transformBlockTo.bind( this );
```

Comment:
If we use something like `expect.poll( editor.getEditedPostContent )`, `this` will be changed. 
Since  `this` in these methods must be an instance of `Editor`, add `bind(this)`.

Reply:
I see! I think this makes sense, thanks! Could you help adding `.bind(this)` to other classes as well? Like `admin` and `requestUtils` etc.